### PR TITLE
feat: v3.0.0 update

### PR DESCRIPTION
**Pull request by srodenhuis**
Run the branch locally `npm run start` and review the contents for:

- inconsistency in formatting
- spelling
- typo's
- correct rebranding
- deprecated information
-

### CODE REVIEW

File: docs/get-started/installation/linode.md [hunk 1]
```diff
@@ -1,98 +1,151 @@
...
+- Create an account for Linode [here](https://cloud.linode.com/)
+
+## Provision a LKE cluster
+
+### Using Cloud Manager
+
+Provision a LKE cluster with the following specs:
 
-## Configure Linode Kubernetes cluster
+- Fill in teh `Cluster Label`
```
Comment (ferruhcihan) [inline docs/get-started/installation/linode.md:21]: Fill in the

File: docs/get-started/labs/create-sealed-secrets.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- `password=helloworld`
+- `username=labs-user`
+
+![Create sealed secret](../../img/create-sealed-secrets.png)
+
+6. Click on `submit`.
+
+7. Click on `Deploy Changes`.
+
+Note that after the secret is created you can not see the values in APL Console anymore:
```
Comment (ferruhcihan) [inline docs/get-started/labs/create-sealed-secrets.md:30]: The secret value will only be visible at the time of creation or once it has been successfully synchronized with the cluster.

**Review by ferruhcihan — COMMENTED** [review]
Some minor changes are needed.

File: docs/for-ops/how-to/clone-apl.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+:::info
+It is advised to keep this file save as it contains passwords and access credentials.
+:::
+
+### Use the values to install a new APL instance
+
+Depending on your use case, you may need to change some settings from the `values.yaml` file before using it to install a new APL instance. The most common use cases:
+
+- Change the `domianSuffix`.
```
Comment (ElderMatt) [inline docs/for-ops/how-to/clone-apl.md:39]: `domianSuffix ` to `domainSuffix `

**Review by ElderMatt — CHANGES_REQUESTED** [review]

**Review by ElderMatt — CHANGES_REQUESTED** [review]

File: docs/get-started/labs/labs-prerequisites.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Harbor
+- Prometheus
+- Loki
+- Grafana
+- Trivy Operator
+
+For the [Using OpenTelemery](use-otel.md) Lab, the Tempo app needs to be enabled together with tracing configured in the `Istio` and `Nginx Ingress` apps.
+
+3. A team called `labs` is created with `Grafana`, `Prometheus` and `Alertmanager` activated.
```
Comment (merll) [inline docs/get-started/labs/labs-prerequisites.md:21]: Are steps 3 and 4 explained somewhere? In the post-installation steps it is briefly outlined how this is done, but regarding and admin user. There is a page "Creating teams" but it is no longer in the TOC.

File: docs/get-started/installation/post-install-steps.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+
+10. Add the user to the required group (otomi-admin).
+
+11. Click `Create`.
+
+12. Choose the `Credentials` tab and then `Set password`.
+
+13. Fill in a password.
+
+14. Optional: Make te password `Temporary`. This requires the user to change the password at the first login.
```
Comment (merll) [inline docs/get-started/installation/post-install-steps.md:85]:
```suggestion
14. Optional: Make the password `Temporary`. This requires the user to change the password at the first login.
```

File: docs/apps/falco.md [hunk 6]
```diff
@@ -4,22 +4,18 @@ title: Falco
 sidebar_label: Falco
 ---
 
-Falco can be used for thread detection. Before activating Falco, please first check which [Driver](https://falco.org/docs/event-sources/kernel/) to use. If you know which driver should be selected, activate Falco, go to the `Values`, select the `Driver` and submit changes. No `Deploy Changes`
+## About
 
-## Configuration values
+Falco can be used for thread detection.
```
Comment (merll) [inline docs/apps/falco.md:9]:
```suggestion
Falco can be used for threat detection. 
```

File: docs/apps/cloudnativepg.md [hunk 7]
```diff
@@ -4,4 +4,6 @@ title: Cloudnative Postgresql
 sidebar_label: Cloudnative Postgresql
 ---
 
-CloudNativePG is used by Otomi to provide Postgresql database for Otomi applications like Harbor and Keycloak. In the values you can configure a storageprovider to store backups in (external) object storage. The backups can be enabled in the settings.
\ No newline at end of file
+## About
+
+CloudNativePG is used by APL to provide Postgresql database for APL applications like Harbor and Keycloak. Configure a storage provider to store backups in (external) object storage. Backups can be enabled in the settings.
```
Comment (merll) [inline docs/apps/cloudnativepg.md:9]:
```suggestion
CloudNativePG is used by APL to provide PostgreSQL databases for APL applications like Harbor and Keycloak. Configure a storage provider to store backups in (external) object storage. Backups can be enabled in the settings.
```

File: docs/apps/apps-overview.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+X.509 certificate management for Kubernetes.
+
+### [Sealed secrets](sealedsecrets.md)
+Sealed Secrets is a Kubernetes Custom Resource Definition Controller which allows you to store even sensitive information in Git repositories.
+
+### [Falco](falco.md)
+Falco is an open source cloud native runtime security tool that makes it easy to consume kernel events, and enrich those events with information from Kubernetes.
+
+### [ExternalDNS](external-dns)
+ExternalDNS synchronizes exposed Kubernetes Services and Ingresses with DNS providers.
```
[the remark this answers is not in the document]
Comment (merll) [reply docs/apps/apps-overview.md:55]: The product name on https://github.com/kubernetes-sigs/external-dns/ is written as ExternalDNS. But we should use it here more consistently.

File: docs/get-started/labs/create-netpols.md [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+env:
+  - name: DATABASE_USER
+    valueFrom:
+      secretKeyRef:
+        name: <psql-cluster-name>-superuser
+        key: username
+  - name: DATABASE_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: <psql-cluster-name>-superuser
```
Comment (merll) [inline docs/get-started/labs/create-netpols.md:111]:
```suggestion
        name: <psql-cluster-name>-app
```

File: docs/get-started/labs/create-netpols.md [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+```yaml
+containerPorts:
+  - name: http
+    containerPort: 80
+    protocol: TCP
+env:
+  - name: DATABASE_USER
+    valueFrom:
+      secretKeyRef:
+        name: <psql-cluster-name>-superuser
```
[the remark this answers is not in the document]
Comment (merll) [reply docs/get-started/labs/create-netpols.md:106]: Both the `-superuser` and the `-app` secret exist, but the former really should not be exposed to applications.

File: docs/get-started/labs/create-netpols.md [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
```
Comment (merll) [inline docs/get-started/labs/create-netpols.md:134]:
```suggestion
        name: <psql-cluster-name>-app
```

File: docs/get-started/labs/create-netpols.md [hunk 12]
```diff
@@ -0,0 +1,238 @@
...
+        key: username
+  - name: DATABASE_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: <psql-cluster-name>-superuser
```
Comment (merll) [inline docs/get-started/labs/create-netpols.md:139]:
```suggestion
        name: <psql-cluster-name>-app
```

File: docs/apps/apps-overview.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (merll) [reply docs/apps/apps-overview.md:55]: I just adjusted the use of this in other places.

File: docs/get-started/labs/labs-prerequisites.md (hunk 4 above)
Comment (srodenhuis) [reply docs/get-started/labs/labs-prerequisites.md:21]: Yes. Both in the post installation steps

**Review by ferruhcihan — APPROVED** [review]
LGTM

File: docs/apps/apps-overview.md [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+
+### [Open Telemetry Operator](otel.md)
+The OpenTelemetry Operator offers a vendor-agnostic implementation on how to receive, process and export telemetry data.
+
+---
+
+### Service Mesh & Proxy
+
+### [Istio](istio.md)
+Istio is an open platform for providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies and aggregate telemetry data.
```
Comment (merll) [inline docs/apps/apps-overview.md:90]:
```suggestion
Istio is a service mesh for providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies and aggregate telemetry data.
```

**Review by ElderMatt — APPROVED** [review]

**Pull request merged**